<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QWidget" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>VColCOSP-Main Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(19, 126, 124);</string>
  </property>
  <widget class="QPushButton" name="pb_loc">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(19, 126, 124);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Localization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_phero">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::disabled {
    border: 4px solid rgb(149, 149, 149);
	color:rgb(149, 149, 149);
	border-radius:10px;
}
QPushButton{
border: 4px solid rgb(19, 126, 124);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>Pheromone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_com">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(19, 126, 124);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Communication</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_vscene">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::disabled {
    border: 4px solid rgb(149, 149, 149);
	color:rgb(149, 149, 149);
	border-radius:10px;
}
QPushButton{
border: 4px solid rgb(19, 126, 124);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>Visual Scene</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_sys_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>621</width>
     <height>81</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="et_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(19, 126, 124);
border-top-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="et_welcome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(19, 126, 124);
border-top-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Welcome to VColCOSP, </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
